Numberdrome.js

```diff
@@ -0,0 +1,50 @@
+'use strict';
+
+class Numberdrome {
+    constructor() {
```

Comment:
use 2 whitespaces instead of 4

Reply:
https://github.com/mate-academy/style-guides/blob/master/javascript.md#whitespace--spaces